ui/app/src/Utils/Api.elm

```diff
@@ -66,7 +66,7 @@ iso8601Time =
 
 baseUrl : String
 baseUrl =
-    "/api/v1"
+    "http://localhost:9093/api/v1"
```

Comment:
Ou, sorry - should not have been included in the commit.

Thought about this as well. But this would mean we need some kind of templating for the index.html file, right?